bgp_sessions_test.go

```diff
@@ -0,0 +1,95 @@
+package packngo
+
+import (
+	"testing"
+)
+
+func TestAccBGPSession(t *testing.T) {
+	skipUnlessAcceptanceTestsAllowed(t)
+
+	c, projectID, teardown := setupWithProject(t)
+	defer teardown()
+
+	hn := randString8()
+	cr := DeviceCreateRequest{
+		Hostname:     hn,
+		Facility:     testFacility(),
+		Plan:         "baremetal_0",
+		ProjectID:    projectID,
+		BillingCycle: "hourly",
+		OS:           "coreos_stable",
+	}
+
+	d, _, err := c.Devices.Create(&cr)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	d, err = waitDeviceActive(d.ID, c)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	bgpSession, _, err := c.BGPSessions.Create(d.ID, CreateBGPSessionRequest{AddressFamily: "ipv4"})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	sessionID := bgpSession.ID
+
+	sessions, _, err := c.Devices.ListBGPSessions(d.ID, nil)
```

Comment:
this still fails with 
```
{"errors":["BGP is not enabled"]}
```

Reply:
Yes, this makes sense, BGP needs to be "enabled" in a project before you can create BGP sessions, so it means we need to make a call to add a bgp config before we create a session. It's an oddity of our API (you can't un-enable bgp on a project after either). It might be worth exploring an API change to make this more intuitive, but for now we should work with the current behavior